prediction/src/algorithms/classify/src/gtr123_model.py

```diff
@@ -288,23 +242,17 @@ def predict(image_itk, nodule_list, model_path="src/algorithms/classify/assets/g
 
     if torch.cuda.is_available():
         casenet = torch.nn.DataParallel(casenet).cuda()
-    # else:
+        # else:
```

Comment:
I'm not sure if the indentation level is now correct?

Reply:
Thanks :)